e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -288,6 +288,35 @@ describe('I18456 Built-in slash commands: common', () => {
         cy.uiWaitUntilMessagePostedIncludes(`You were added to the channel by @${user1.username}`);
     });
 
+    it('MM-T661 /invite extra white space before @ in DM or GM', () => {
+        const user = userGroup[6];
+        const userToInviteGM = userGroup[5];
+        const userToInviteDM = userGroup[4];
+
+        cy.apiAddUserToChannel(testChannel.id, user.id);
+        loginAndVisitDefaultChannel(user, `${team1.name}/channels/${testChannel.name}`);
+        cy.get('#postListContent', {timeout: TIMEOUTS.HALF_MIN}).should('be.visible');
+
+        // # In a GM use the /invite command to invite a user to a channel you have permission to dd them to but place extra white space before the username
```

Comment:
I've corrected the typo in test case.
```suggestion
        // # In a GM use the /invite command to invite a user to a channel you have permission to add them to but place extra white space before the username
```

Reply:
Whoops! Fixed now.